docs/core/config/auth/databases/ldap.md

```diff
@@ -99,12 +109,17 @@ lowercase the username via [[setting,auth_username_format,%Lu]].
 
 A typical configuration would look like:
 
+::: code-group
```

Comment:
These code-groups probably aren't useful when there is just one file?

Reply:
I made it intentionally because it shows in a very readable manner which is the pertinent configuration file.

I think this is useful because the configuration location has changed from the former xxxx.ext files and this can help the users to notice it

Reply:
These seem to have been done differently recently, I think it's been:

```
```[dovecot.conf]
```

Reply:
Oh, that's how you have it there also. Hmm. I think Karl was removing these single-file code-groups also at some point.